crates/winscard-rs/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "winscard-rs"
```

Comment:
suggestion: Let’s call the crate `winscard` without the `-rs` suffix. The `-rs` suffix is nice to have in the name of repositories to differentiate from other popular crates with similar names, but we already know that all crates are Rust code.
https://rust-lang.github.io/api-guidelines/naming.html#casing-conforms-to-rfc-430-c-case

Reply:
done